src/components/external-source/ExternalSourceManager.svelte

```diff
@@ -0,0 +1,1411 @@
+                      </div>
+                    {:else}
+                      <div class="st-typography-label empty-state">No external source types matching filter</div>
+                    {/if}
+                  </div>
+                  <div class="list-buttons menu-border-top">
+                    <button
+                      class="st-button secondary list-button"
+                      id="source-filters-select-all"
+                      on:click={selectFilteredValues}
+                    >
+                      Select {filteredValues.length}
+                      {#if filteredValues.length === 1}
+                        {'external source type'}
+                      {:else}
+                        {'external source types'}
+                      {/if}
+                    </button>
+                    <button class="st-button secondary list-button" on:click={unselectFilteredValues}
+                      >Unselect all</button
+                    >
+                  </div>
+                </div>
+              </Menu>
+            </div>
+          </div>
+        </slot>
+        <slot name="right">
+          <button
+            name="manage-derivation-groups"
+            class="st-button secondary"
+            on:click|stopPropagation={onManageGroupsAndTypes}
+            use:tooltip={{
+              content: 'Manage derivation groups, external source types, and external event types.',
+              placement: 'top',
+            }}
+          >
+            Manage Groups and Types
+          </button>
+        </slot>
+      </svelte:fragment>
+      <svelte:fragment slot="body">
+        {#if $externalSources.length}
+          <SingleActionDataGrid
+            {columnDefs}
+            {hasDeletePermission}
+            itemDisplayText="External Source"
+            items={filteredExternalSources}
+            {user}
+            getRowId={getRowIdExternalSourceSlim}
+            on:rowClicked={({ detail }) => selectSource(detail.data)}
+            bind:selectedItemId={selectedSourceId}
+          />
+        {:else}
+          <p>No external sources matching the selected external source type(s).</p>
+        {/if}
+      </svelte:fragment>
+    </Panel>
+
+    <!--
+      -- Tooltip only applies to timeline, but to ensure it resets and doesn't unexpectedly show up when the timeline isn't selected,
+      --      had to be moved here and had to add calls to reset
+      --      in various locations.
+      -->
+    <Tooltip
+      bind:this={eventTooltip}
+      {mouseOver}
+      interpolateHoverValue={false}
+      hidden={!!selectedSource && !showExternalEventTimeline}
+      resourceTypes={[]}
+    />
+
+    {#if selectedSource}
+      <CssGridGutter track={1} type="row" />
+
+      <!-- External Event Table/Timeline -->
+      <Panel padBody={true}>
+        <svelte:fragment slot="header">
+          <slot name="left">
+            <SectionTitle><Balloon />External Events</SectionTitle>
```

Comment:
Why is the Balloon icon used to represent External Events?

Reply:
Party <-> Event connection?

Reply:
This could be open to discussion.

Reply:
Was this ever resolved? The balloon still doesn't feel quite right to me thematically.

Reply:
Let me check in with Lyle!